src/synthesizer/parametric/sf_hist.py

```diff
@@ -118,9 +120,29 @@ def calculate_sfh(self, t_range=(0, 10**10), dt=10**6):
 
         return t, sfh
 
+    def plot_sfh(self, t_range=(0, 10**10), dt=10**6, log=False):
+        """
+        Create a quick plot of the star formation history.
+
+        Args:
+            t_range (tuple, float)
+                The age limits over which to calculate the SFH.
+            dt (float)
+                The interval between age bins.
+        """
+
+        t, sfr = self.calculate_sfh(t_range=t_range, dt=dt)
+        plt.plot(t, sfr)
```

Comment:
Would this not be better done as a step plot given the gridded nature of the SFH?

Reply:
Not here right, `calculate_sfh` doesn't integrate across a bin but gets the SFR at the times given.

Reply:
Ah yea, forgot that.